content/en/continuous_integration/search/_index.md

```diff
@@ -40,7 +40,7 @@ You can access high-level accumulation and trends, including:
 
 To see your pipelines, navigate to [**CI** > **Pipelines**][101].
 
-The [Pipelines page][101] shows aggregate stats for the default branch of each pipeline over the selected time frame, as well as the status of the latest pipeline execution. Use this page to see all your pipelines and get a quick view of their health. The Pipelines page shows metrics for the _default_ branch, usually named something like `main` or `prod`.
+The [Pipelines page][101] shows aggregate stats of your pipelines as well as the status of the latest pipeline execution. For pipelines with Git information, it only shows stats for the default branch of each pipeline over the selected time frame. The _default_ branch is usually named something like `main` or `prod`. Use this page to see all your pipelines and get a quick view of their health.
```

Comment:
shows aggregate stats for the default branch of each pipeline over the selected time frame, as well as the status of the latest pipeline execution. Use this page to see all your pipelines and get a quick view of their health. Only pipelines with Git information associated to the _default_ branch (usually named something like `main` or `prod`), as well as pipelines without any Git information, are displayed on this page.